authority/provisioner/controller.go

```diff
@@ -192,3 +198,10 @@ func SanitizeSSHUserPrincipal(email string) string {
 		}
 	}, strings.ToLower(email))
 }
+
+func (c *Controller) GetPolicy() *policyEngine {
+	if c == nil {
+		return nil
+	}
+	return c.policy
+}
```

Comment:
We can make it public like the rest of fields.

Reply:
We could, but wouldn't that partially undo the benefit of containing it in the `Controller`?

Reply:
The reason why I put all the fields in the controller public was that I was thinking that we can switch it over for another implementation, but adding authority options `authority.With...` to configure the `provisioner.Config` made more sense back then. I finally left everything public to make it "easier" to use if necessary.